src/main/java/seedu/hirehub/logic/commands/StatusCommand.java

```diff
@@ -28,70 +21,72 @@ public class StatusCommand extends Command {
     public static final String COMMAND_WORD = "status";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates recruitment status of a candidate"
-            + " existing in the address book to one of the five status:\n"
+            + " existing in the list of applications to one of the five status:\n"
             + "PRESCREEN, IN_PROGRESS, WAITLIST, ACCEPTED, REJECTED\n"
-            + "Parameters: INDEX (must be a positive integer) STATUS \n"
-            + "Example: " + COMMAND_WORD + " 2 ACCEPTED";
+            + "Parameters: e/EMAIL ti/jobTitle s/Status \n"
+            + "Example: " + COMMAND_WORD + " e/acekhoon@gmail.com ti/Quantitative Trader s/ACCEPTED";
 
-    public static final String MESSAGE_STATUS_PERSON_SUCCESS = "Status of Candidate Number %1$d Successfully"
-            + " Updated to %2$s";
+    public static final String MESSAGE_STATUS_PERSON_SUCCESS = "Status of Candidate Successfully"
+            + " Updated to %1$s";
 
-    public static final String MESSAGE_DUPLICATE_PERSON = "This candidate with identical recruitment status "
-            + "already exists in the address book or this candidate's status is already set to %1$s.";
+    public static final String MESSAGE_APPLICATION_NOT_FOUND = "Provided candidate who applied for Job \"%1$s\" not"
+            + " found in the list of available applications";
+
+    public static final String MESSAGE_DUPLICATE_STATUS = "This candidate with identical recruitment status %1$s "
+            + "already exists in the application list";
 
     public static final String STATUS_CANNOT_BE_EDITED = "Status of candidates cannot be edited via edit method.\n"
             + "Please use status command instead in order to update recruitment status for candidates.";
```

Comment:
Would it be better for this error message to be in EditCommand instead?

Reply:
okay sure